static/css/perdiem.css

```diff
@@ -272,7 +272,7 @@ a.nav-color {
     color: #fff;
 }
 
-form > input[type=submit] {
+form > input [type=submit] {
```

Comment:
This change is not what I meant. For each CSS attribute for this selector, there is a attribute name and then colon and then value. There should be a space between the colon and the value. ie. it should look like:
```css
form > input[type=submit] {
    display: block;
    margin: 0 auto;
}
```